tests/utils.py

```diff
@@ -21,6 +23,16 @@
 from smdebug.exceptions import TensorUnavailableForStep
 
 
+class Timer(ContextDecorator):
+    def __enter__(self):
+        self.start_time = time.perf_counter()
```

Comment:
qq: Why `time.perf_counter` instead of `time.time`?

Reply:
`time.perf_counter`: https://docs.python.org/3/library/time.html#time.process_time